File: Rado/part365/src/app/model/userCount.ts
export class UserCount
{
    carCount!: number;
    busCount!: number;
    partCarCount!: number;
    partBusCount!: number;
    rimCount!: number;
    tyreCount!: number;
    rimWithTyreCount!: number;

    total() {
        return this.carCount + this.busCount+ this.partCarCount + this.partBusCount+ this.rimCount+ this.tyreCount + this.rimWithTyreCount;

    }
}

File: Models/Models/UserCount.cs
using Models.Models.Authentication;

namespace Rado.Models
{
    public class UserCount
    {
        public User user;
        public int carCount { get; set; }
        public int busCount { get; set; }
        public int partCarCount { get; set; }
        public int partBusCount { get; set; }
        public int rimCount { get; set; }
        public int tyreCount { get; set; }
        public int rimWithTyreCount { get; set; }
    }
}
